src/test/java/unit/SshKeyPair.java

```diff
@@ -18,54 +22,70 @@ public class SshKeyPair extends SingleWorkspaceUnit {
   private TestUser testUser = TestUser.chooseTestUser();
   private TestUser testUser2 = TestUser.chooseTestUserWhoIsNot(testUser);
 
+  // TODO (PF-1497): Remove this once ECM is deployed to all environment in verily.
+  private static final List<String> AVAILABLE_VERILY_ECM_SERVER = ImmutableList.of("verily-devel");
+
```

Comment:
Did you look at a `@DisableIf` and `@DisabledIfEnvironmentVariable`? If we had an environment variable for the server (and I bet we do somewhere), we could use that at the test level.

Reply:
i didn't know. changed to DisabledIfEnvironmentVariable instead.